microprot/scripts/split_search.py

```diff
@@ -525,7 +536,6 @@ def mask_sequence(hhsuite_fp, fullsequence_fp, subsequences_fp=None,
 
     # collect gaps between positive hits
     subseqs_neg = report_uncovered_subsequences(subseqs_pos, str(p),
-                                                min_subseq_len=
                                                 min_fragment_length)
```

Comment:
because I was not able to comply to pep8 leaving that in AND it is not necessary to address the variable by name. It will be assigned to min_subseq_len if it is the third argument.